train/train_utils/logger.py

```diff
@@ -2,6 +2,7 @@
 
 import pathlib
 import logging
+from torch.utils.tensorboard import SummaryWriter
```

Comment:
See [here](https://google.github.io/styleguide/pyguide.html#22-imports) on imports. Only import packages and modules.